cluster-up-sha.txt

```diff
@@ -1 +1 @@
-af81cad31fcd27eae71a9d5b9d3263cb36919db2
+741fbb38ee8ef10ef9a414ce4651268296e46161
```

Comment:
This  is strange. That seems to be the only file which is changed from the kubevirtci bump. Is that a left-over?

Reply:
I called bump/sync-kubevirtci and It only updated this hash. Maybe somebody forgot to bump the hash.

Reply:
ack.